2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_setup.py:_flush():76] Current SDK version is 0.15.2
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_setup.py:_flush():76] Configure stats pid to 1160594
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_setup.py:_flush():76] Loading settings from /home/veselovs/.config/wandb/settings
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_setup.py:_flush():76] Loading settings from /scratch/venia/socialgpt/venia_worker_vs_gpt/src/wandb/settings
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'finetune_model.py', 'program': '/scratch/venia/socialgpt/venia_worker_vs_gpt/src/finetune_model.py'}
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_init.py:_log_setup():507] Logging user logs to /scratch/venia/socialgpt/venia_worker_vs_gpt/src/wandb/run-20230524_185233-21x4tqzy/logs/debug.log
2023-05-24 18:52:33,053 INFO    MainThread:1160594 [wandb_init.py:_log_setup():508] Logging internal logs to /scratch/venia/socialgpt/venia_worker_vs_gpt/src/wandb/run-20230524_185233-21x4tqzy/logs/debug-internal.log
2023-05-24 18:52:33,054 INFO    MainThread:1160594 [wandb_init.py:init():547] calling init triggers
2023-05-24 18:52:33,054 INFO    MainThread:1160594 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'ckpt': 'intfloat/e5-base', 'batch_size': 2, 'lr': 2e-05, 'num_epochs': 10, 'weight_decay': 0, 'train_size': 500, 'use_augmented_data': True, 'approach': 'simple', 'augmentation_model': 'gpt-3.5-turbo'}
2023-05-24 18:52:33,054 INFO    MainThread:1160594 [wandb_init.py:init():596] starting backend
2023-05-24 18:52:33,054 INFO    MainThread:1160594 [wandb_init.py:init():600] setting up manager
2023-05-24 18:52:33,063 INFO    MainThread:1160594 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-05-24 18:52:33,069 INFO    MainThread:1160594 [wandb_init.py:init():606] backend started and connected
2023-05-24 18:52:33,080 INFO    MainThread:1160594 [wandb_init.py:init():700] updated telemetry
2023-05-24 18:52:33,081 INFO    MainThread:1160594 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2023-05-24 18:52:33,453 INFO    MainThread:1160594 [wandb_run.py:_on_init():2177] communicating current version
2023-05-24 18:52:33,530 INFO    MainThread:1160594 [wandb_run.py:_on_init():2186] got version response upgrade_message: "wandb version 0.15.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-05-24 18:52:33,530 INFO    MainThread:1160594 [wandb_init.py:init():787] starting run threads in backend
2023-05-24 18:52:35,341 WARNING MainThread:1160594 [wandb_init.py:init():1172] interrupted
Traceback (most recent call last):
  File "/scratch/venia/python/miniconda3/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1150, in init
    run = wi.init()
  File "/scratch/venia/python/miniconda3/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 799, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/scratch/venia/python/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/scratch/venia/python/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/scratch/venia/python/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/scratch/venia/python/miniconda3/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/scratch/venia/python/miniconda3/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2023-05-24 18:52:42,022 WARNING MsgRouterThr:1160594 [router.py:message_loop():77] message_loop has been closed
